src/components/updateItem.js

```diff
@@ -0,0 +1,252 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import * as creators from "../state/actionCreators";
+import {
+  Form,
+  Input,
+  Icon,
+  Button,
+  message,
+  Upload
+} from "antd";
+
+import "../less/index.less";
+import axios from "axios";
+import AxiosAuth from "./Auth/axiosWithAuth";
+import history from "../history";
+
+function UpdateItem(props) {
+  const [item, setItem] = useState([]);
+  console.log(item)
+  const itemId = props.match.params.id;
+  const productURL =
+  `https://shopping-cart-eu3-staging.herokuapp.com/api/store/products/${itemId}`;
+  useEffect(() => {
+    AxiosAuth()
+      .get(
+        `https://shopping-cart-eu3-staging.herokuapp.com/api/store/products/${itemId}`
+      )
+      .then(res => {
+        setItem(res.data);
+      });
+  }, [itemId]);
+
+
+const test = item && item.images && item.images.length && item.images.map(i=>{
+  return i
+})
+const myArray = test && test.length && test.map((str, index) => ({ url: str, id: index + 1 , name:'image.png'}));
+console.log(myArray)
```

Comment:
Can we remove console.logs before merging?

Reply:
Done